sunpy/map/tests/test_maputils.py

```diff
@@ -175,3 +196,46 @@ def test_data_at_coordinates(aia171_test_map, aia_test_arc):
     intensity_along_arc = aia171_test_map.data[y, x]
     np.testing.assert_almost_equal(data[0], intensity_along_arc[0], decimal=1)
     np.testing.assert_almost_equal(data[-1], intensity_along_arc[-1], decimal=1)
+
+
+def test_contains_solar_center(aia171_test_map, all_off_disk_map, all_on_disk_map, straddles_limb_map, sub_smap):
+    assert contains_solar_center(aia171_test_map)
+    assert not contains_solar_center(all_off_disk_map)
+    assert not contains_solar_center(all_on_disk_map)
+    assert not contains_solar_center(straddles_limb_map)
+    assert not contains_solar_center(sub_smap)
+
+
+def test_verify_coordinate_helioprojective(aia171_test_map, all_off_disk_map, all_on_disk_map, straddles_limb_map,
+                                           sub_smap, non_helioprojective_map, non_helioprojective_skycoord):
+    # These should be helioprojective.
+    _verify_coordinate_helioprojective(aia171_test_map.coordinate_frame)
+    _verify_coordinate_helioprojective(all_off_disk_map.coordinate_frame)
+    _verify_coordinate_helioprojective(all_on_disk_map.coordinate_frame)
+    _verify_coordinate_helioprojective(straddles_limb_map.coordinate_frame)
+    _verify_coordinate_helioprojective(sub_smap.coordinate_frame)
+    # These are not.
+    with pytest.raises(ValueError, match=r".* must be in the Helioprojective frame."):
+        _verify_coordinate_helioprojective(non_helioprojective_map.coordinate_frame)
+    with pytest.raises(ValueError, match=r".* must be in the Helioprojective frame."):
+        _verify_coordinate_helioprojective(non_helioprojective_skycoord)
+
+
+def test_functions_raise_non_frame_coords(non_helioprojective_map):
```

Comment:
```suggestion
def test_functions_raise_non_frame_coords(non_helioprojective_skycoord):
```
The wrong fixture is specified here.  The tests pass because passing in a totally inappropriate type (a function) gives the same `ValueError` as passing in the fixture with the incorrect frame.

Reply:
What I get for not matching error messages.